docs/pricing/billing-faqs.mdx

```diff
@@ -21,3 +21,7 @@ After updating or resolving your payment method issue that occurred during the a
 1. Wait for our system to re-attempt the charge for the invoice, it usually takes up to 24 hours. If it does not happen after 24 hours, please contact [Support](support@sourcegraph.com) to resolve the issue.
 1. In the **Workspace settings > Billing** page, click on the **Manage payments** button, which takes you to the Stripe Customer Portal, and pay the invoice there. Our system will reconcile your payment within 24 hours, if it does not reconcile after 24 hours, please contact [Support](support@sourcegraph.com) to resolve the issue.
 1. Contact [Support](support@sourcegraph.com) to request re-attempt the charge for the invoice using the payment method on file.
+
+## Are there any refunds for the subscription?
+
+We don't offer refunds, but if you have any queries regarding the Enterprise Starter, please write to support@sourcegraph.com, and we'll help resolve the issue.
```

Comment:
This is not aligned with our terms IMO, see section 8.3 of https://sourcegraph.com/terms/cloud

Reply:
Hmm, these are the ~exact words copied from Cody Pro

![image](https://github.com/user-attachments/assets/0c934a4a-d392-47d3-9e12-00ea30359009)

Reply:
God damn it, our dark background page design literally killed my eyes. I think it actually aligns?

> All fees and charges are earned upon receipt by us and are nonrefundable unless negotiated otherwise

Contacting support is applicable for "negotiated otherwise"? BTW this repo and our comments here are public.

Reply:
I think the "or (b) as required by applicable law." section is missing here for me.

Maybe we could link to terms? (deeplinking is sadly broken there) 

Not a lawyer but this statement feels incomplete to me. Won't block here but I think it could use some wordsmithing haha

Reply:
>deeplinking is sadly broken there

Already asked for a fix! https://sourcegraph.slack.com/archives/CNC4YAL1E/p1739389461245829 so we can still do deep link and wait for it to be fixed.

Reply:
Let's wait for legal response first!

Reply:
[Nudged again on the Legal thread](https://sourcegraph.slack.com/archives/C01GJFE26BX/p1739819021184179?thread_ts=1739294636.840439&cid=C01GJFE26BX) for a suggested wording. If we couldn't get a effective response by tomorrow, I am gonna merge as-is first.